### ISSUE TITLE
Refresh activity statistics

### ISSUE THREAD

**Issue by asorli** [jimmykane/quantified-self#191]
After "Refresh activity statistics" some activity parameters ("Recovery time", "Energy", "Peak EPOC", "Total Traininig Effects") disappears. 

![image](https://user-images.githubusercontent.com/34653992/76200309-adda5100-61f1-11ea-8121-b7eec71f624e.png)
![image](https://user-images.githubusercontent.com/34653992/76200324-b763b900-61f1-11ea-99f3-aa6cfb190cc9.png)

BR,
Andrej

**Comment by jimmykane** [comment]
That is correct. That is because all data are wiped and then recreated but propertiary data go away. 

you think that should be improved?

**Comment by asorli** [comment]
I think it would be better to preserve proprietary data during update.
